yast/src/modules/SMTData.rb

```diff
@@ -64,8 +63,6 @@ def main
       @SCCcredentials_file = "/etc/zypp/credentials.d/SCCcredentials"
       @NCCcredentials_file = "/etc/zypp/credentials.d/NCCcredentials"
```

Comment:
There is no migration for NCCcredentials?
I get that the 'NCC' here is pure cosmetic (since we have ApiType config), but for future reference, is it correct to look for both of these files when looking for credentials even when the only available ApiType is SCC?